python/artm/regularizers.py

```diff
@@ -588,3 +589,36 @@ def __init__(self, name=None, tau=1.0, topic_names=None,
                                       config=config,
                                       topic_names=topic_names,
                                       alpha_iter=alpha_iter)
+
+
+class BitermsPhiRegularizer(BaseRegularizerPhi):
```

Comment:
This is my mistake, I fixed it and added the note about it into the instructions in our wiki.

I'll add test for this regularizer today or tomorrow.

16.06.2016, 21:54, "Oleksandr Frei" <notifications@github.com>:
> In python/artm/regularizers.py:
>
>> @@ -588,3 +589,36 @@ def __init__(self, name=None, tau=1.0, topic_names=None, > config=config, > topic_names=topic_names, > alpha_iter=alpha_iter) > + > + > +class BitermsPhiRegularizer(BaseRegularizerPhi):
>
> At the top of regularizers.py file there is a variable called __ALL__. Either we should remove it (if its not used), or add there new regularizer.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.